.changeset/twenty-pianos-invent.md

```diff
@@ -0,0 +1,5 @@
+---
+"@justeattakeaway/pie-chip": minor
```

Comment:
If you wanted to, you could use patch instead of minor here (as it's a small fix). Not a biggie though 😆

Reply:
i will keep that in mind, thanks for flagging it 🥇